web/views/tracker_view.ex

```diff
@@ -112,7 +112,7 @@ defmodule Healthlocker.TrackerView do
                   |> Map.get(:hours_slept)
       List.insert_at(list, 0, hours_slept)
     else
-      List.insert_at(list, 0, 0)
+      List.insert_at(list, 0, "null")
```

Comment:
Yeah, got it. but I don't _understand_ the reason for switching from 0 values i.e. always the same `type`.